packages/cli/test/commands/fix/__snapshots__/fix.test.ts.snap

```diff
@@ -0,0 +1,333 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Command: fix "base_ts_app" fixture > fix directory with src arg 1`] = `
+"info:    @rehearsal/fix<test-version>
+[STARTED] Initialize
+[SUCCESS] Initialize
+[STARTED] Infer Types
+[DATA] processing file: /vitest.config.ts
```

Comment:
Is this expected?

Reply:
You saying `vitest.config.ts` <<<< this vitest file?